CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -634,15 +628,24 @@ private void DestroyVertexBuffer() {
     
     // ========================> Push constants <======================== \\
     
-    private void CreatePushConstants() {
+    private int CreatePushConstants() {
         // Initialise push constants to identity mtx
-        pushConstants = memAlloc(VertexUniformBufferObject.SIZEOF);
-        for (int iRow = 0; iRow < 4; ++iRow) {
-            for (int iCol = 0; iCol < 4; ++iCol) {
-                pushConstants.putFloat(IDENTITY_44F.get(iRow, iCol));
+        try {
+            pushConstants = memAlloc(VertexUniformBufferObject.SIZEOF);
+            for (int iRow = 0; iRow < 4; ++iRow) {
+                for (int iCol = 0; iCol < 4; ++iCol) {
+                    pushConstants.putFloat(IDENTITY_44F.get(iRow, iCol));
+                }
             }
+            pushConstants.flip();
         }
-        pushConstants.flip();        
+        catch(Exception ex) {
+            // Out of memory error
+            cvkDevice.GetLogger().severe("Out of memory allocating push constants %s", ex.toString());
```

Comment:
CVKGraphLogger has a LogException that prints your message, the exception message and the exception call stack.

Reply:
Changed